phase1/gce/do

```diff
@@ -13,6 +13,12 @@ CLUSTER_NAME=$(jq -r '.phase1.cluster_name' ${CONFIG_FILE})
 CLUSTER_DIR=clusters/${CLUSTER_NAME}
 TMP_DIR=${CLUSTER_DIR}/.tmp
 
+# If 'account.json' is missing, try to use the file defined in 'GOOGLE_APPLICATION_CREDENTIALS'
+if [[ ! -e account.json ]] && [[ ! -z "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]] &&
+   [[ -e "${GOOGLE_APPLICATION_CREDENTIALS}" ]]; then
+  cp "${GOOGLE_APPLICATION_CREDENTIALS}" account.json
```

Comment:
Does a soft link work instead of a copy? I'm worried about copying credentials all over the place.

Reply:
the version in the runner was doing that - probably the better solution.
will update in a bit.